models/repo.go

```diff
@@ -696,6 +696,18 @@ func (repo *Repository) getReviewers(e Engine, doerID, posterID int64) (users []
 	} else {
 		users, err = repo.getReviewersPublic(x, doerID, posterID)
 	}
+
+	if repo.Owner.IsOrganization() {
+		teams, err = GetTeamsWithAccessToRepo(repo.OwnerID, repo.ID, AccessModeRead)
+		if err != nil {
+			return nil, nil, err
+		}
+
+		for _, team := range teams {
+			team.ID = -team.ID
```

Comment:
I think func in models should return id's as they are ... if needed convert it where you need it

Reply:
Done